Packs/Redmine/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Redmine
+
+- Updated the Docker image to: *demisto/python3:3.10.14.92207*.
+- Fixed an issue with tracker_id argument to support custom.
+- Fixed an issue with status_id argument to support custom.
+- Fixed an issue with priority_id argument to support custom.
+- Updated the status_id arg in get-issue-list command to status so it will handle both an ID and a predefined status.
```

Comment:
```suggestion
- Fixed an issue with the tracker_id argument to support custom tracker IDs.
- Fixed an issue with the status_id argument to support custom status IDs.
- Fixed an issue with the priority_id argument to support custom priority IDs.
- Updated the status_id argument in the get-issue-list command to status so it will handle both an ID and a predefined status.
```